toptal/picasso
# feat(icon): add career icons

[PRO-X]

### Description

Add Career icons

### How to test

- `yarn start`
- Open [icons page](http://localhost:9001/?path=/story/components-icons--icons) and ensure Career icon looks good

### Screenshots

<img width="319" alt="Screen Shot 2021-01-12 at 16 47 19" src="https://user-images.githubusercontent.com/14301985/104322302-d871f600-54f5-11eb-8019-b42b0a494bde.png">

### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run danger` - Danger checks
- `@toptal-bot run lint` - Run linter
- `@toptal-bot run test` - Run jest
- `@toptal-bot run build` - Check build
- `@toptal-bot run test:visual` or `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>
